architecture/additional_concepts/networking.adoc

```diff
@@ -92,9 +92,18 @@ routing for its endpoints. Service DNS can still be used and responds with
 multiple A records, one for each pod of the service, allowing the client to
 round-robin between each pod.
 
+[[network-plugins]]
+
+== Network Plugins
+
+{product-title} supports the same plugin model as Kubernetes for networking pods. 
+ifdef::openshift-enterprise,openshift-origin[]
+The following network plugins are currently supported by {product-title}.
```

Comment:
The idea is the plugin will be supported by Nuage completely. The install steps are listed so users know  what playbook parameters they need in their nodes file in order to use Nuage as a plugin. In no way is this meant to put the support onus on RH as far as installing openshift with Nuage is concerned. If there are any other questions/concerns, please let us know.